packages/config/config/devices/0x5254/zxt-800.json

```
@@ -8,6 +8,11 @@
 			"productType": "0x0004",
 			"productId": "0x8492",
 			"zwaveAllianceId": 4788
```

This ID belongs to the entry you just added. Not sure why it is assigned to the existing one.
```suggestion
			"productId": "0x8492"
```